### PR TITLE
fix: broken artist screen tests

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [] 

### Description

This PR comes to fix the broken tests in the artist screen. After some investigation, I realized that this was broken due to returning the same interface twice which led to overwriting the previously returned mocked values of `Me`

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/setupJest.tsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ import { NativeModules } from "react-native"
 // @ts-expect-error
 import mockSafeAreaContext from "react-native-safe-area-context/jest/mock"
 import track, { useTracking } from "react-tracking"
+require("@shopify/flash-list/jestSetup")
```
Comment (gkartalis) [inline src/setupJest.tsx:19]: 👀 why do we need this?

File: src/app/Scenes/Artist/Artist.tsx [hunk 2]
```diff
@@ -85,22 +85,29 @@ export const Artist: React.FC<ArtistProps> = (props) => {
...
-      internalID: artistAboveTheFold.internalID,
-      slug: artistAboveTheFold.slug,
-      artists: [{ name: artistAboveTheFold.name ?? null }],
-      title: artistAboveTheFold.name!,
-      href: artistAboveTheFold.href!,
-      currentImageUrl: artistAboveTheFold.coverArtwork?.image?.url ?? undefined,
-    })
+    if (
+      artistAboveTheFold.name &&
+      artistAboveTheFold.name &&
```
Comment (gkartalis) [inline src/app/Scenes/Artist/Artist.tsx:90]:
```suggestion
      artistAboveTheFold.name &&
```

**Review by gkartalis — COMMENTED** [review]
thanks for the extra fixes ❤️ left two comments

File: src/setupJest.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/setupJest.tsx:19]: this comes as from flashlist docs actually - https://shopify.github.io/flash-list/docs/testing/

Comment (MounirDhahri) [reply src/setupJest.tsx:19]: I am not sure when this will hurt us but I guess it doesn't hurt to add it here since they recommend us to do it

**Review by olerichter00 — DISMISSED** [review]
LGTM

**Review by brainbicycle — APPROVED** [review]
Looks good!

**Pull request merged**